src/main/webapp/resources/css/pharmacytoken.css

```
@@ -1,14 +1,14 @@
 @media print{
     .posbill{
-        position: relative!important;  
-        margin-top: -2cm!important;       
+        position: relative!important;
+        margin-top: -2cm!important;
         width: 100%;
         padding: 0px!important;
         margin: 0px!important;
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)